core/core.go

```diff
@@ -106,7 +106,7 @@ func (e Extension) String() string {
 }
 
 func (e Extension) BuildDir() string {
-	return filepath.Join(".", e.Development.RootDir, e.Development.BuildDir)
+	return path.Join(".", e.Development.RootDir, e.Development.BuildDir)
```

Comment:
Did `path.Join` work better on windows?

Reply:
Yes, `path` works on both *nix and Windows, `filepath` does not.